linker-interface/shared/src/main/scala/org/scalajs/linker/interface/CheckedBehavior.scala

```diff
@@ -24,4 +24,14 @@ object CheckedBehavior {
   case object Compliant extends CheckedBehavior
   case object Fatal extends CheckedBehavior
   case object Unchecked extends CheckedBehavior
+
+  private[interface] implicit object CheckedBehaviorFingerprint
+    extends Fingerprint[CheckedBehavior] {
+      override def fingerprint(behavior: CheckedBehavior): String =
```

Comment:
I believe the indentation is wrong here:
https://github.com/scala-js/scala-js/blob/master/CODINGSTYLE.md#class-declaration

`extends` should have 4 spaces, members only 2.

This applies elsewhere as well.

Reply:
Fixed in 5628647d30f13ee91950c380e1bbf5ea597acd01